src/cli/cli.cpp

```diff
@@ -1643,9 +1683,7 @@ otError Interpreter::ProcessEidCache(uint8_t aArgsLength, char *aArgs[])
     for (uint8_t i = 0;; i++)
     {
         SuccessOrExit(otThreadGetNextCacheEntry(mInstance, &entry, &iterator));
-
-        OutputIp6Address(entry.mTarget);
-        OutputLine(" %04x", entry.mRloc16);
+        PrintEidCacheEntry(entry);
```

Comment:
```suggestion
        OutputEidCacheEntry(entry);
```
nit

Reply:
I suggested the same below (noticed this afterwards 😃)